.github/workflows/scan.yaml

```
@@ -16,6 +20,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: "0"
+          submodules: ${{ git.submodules }}
```

Per comment above, this would be changed to:

```suggestion
          submodules: ${{ inputs.submodules }}
```